CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ Given a version number MAJOR.MINOR.PATCH, increment:
 
 
 ## [Unreleased]
+### Added
+- PublicKey.toCompressed() function to generate compressed public key
+- PublicKey.fromCompressed() function to recover a compressed public key
```

Comment:
Talvez assim?
- PublicKey.toCompressed() function to dump a public key in compressed format
- PublicKey.fromCompressed() function to read a public key in compressed format